Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+- Encourage participation and mentorship in wider community events such as hackathons or mentorships.
+- Appointing representatives to work with other open source or open standards communities as needed.
+
+## Composition
+
+The TSC is intended to be a representation of Zowe Committers, as well as other architects, subject matter experts, and members of the extended Zowe community. While any of these individuals may participate in the TSC, there will be a defined group of TSC voting members that will vote on actions of the TSC within the roles of the TSC outlined above.
+
+The initial voting members of the TSC will consist of representatives from the Zowe Leadership Committee (ZLC) as well as the current lead or representative from each squad, as defined below. Where the squad lead is also one of the ZLC representative this position should be filled by another squad SME.
+
+  - Joe Winchester (ZLC representative)
+  - Sean Grady (ZLC representative)
+  - Mark Ackert (ZLC representative)
+  - _TBD_ - (Zowe App Framework/Desktop squad)
+  - Mike Bauer (Zowe CLI Squad)
+  - Fernando Rijo Cedeno (Zowe Explorer squad)
+  - Elliot Jalley (Zowe API Squad)
+  - _TBD_ (Zowe Systems - CI/CD squad)
+  - Nan Nan Li (Zowe Documentation squad)
+  - _TBD_ (Zowe Onboarding squad)
+
+The TSC voting members will have no more than one-third (33%) of voting members affiliated with the same employer. In the case where a TSC voting member has an employment change which violates this requirement, the TSC voting member should be replaced by the body in which thier status is provided.
+
+The TSC voting members may adjust the makeup of the TSC voting members after the initial year with approval of the current Zowe Committers. It is expected that any makeup changes would look to align with the growing technical community structure to ensure equitable representation. Any TSC voting member elections will be facilated by the Open Mainframe Project staff to ensure vendor-neutrality. 
+
+TSC voting members are expected to regularly participate in TSC meetings and other activities. TSC voting members may voluntarily resign at any time, in which case the squad through which the voting member was elected will be responsible for providing a new voting member. In the exceptional situation where the TSC determines a voting member is acting in a manner detrimental to the Open Mainframe Project or Zowe project, the other TSC voting members will consult with Open Mainframe Project staff to determine a course of action.
+
+## Operations and Processes
+
+It is expected that the TSC will establish it's general operations and policies, clearly defined in the [GOVERNANCE.md](../GOVERNANCE.md), but in general the TSC will do the following.
+
+### Chairperson and Secretary
+
+The TSC will elect a chairperson amongst the TSC voting members which will have the responsibilities and duties of...
```

Comment:
Before anyone could be the chairperson, but it's probably best for this to be of voting members only

Reply:
I actually like the idea that a non-voting member could be the chairperson, as it's a different skill set (organization versus SME) so I'll amend it to relax the criteria.  Good call

Reply:
We've tended to not go down this route, as there isn't a good mechanism for qualifying and selecting this role unless it's amongst voting members.